common/scheme_plugin.go

```diff
@@ -148,32 +206,76 @@ func (s *SchemeRPC) GetFormat() AttestationFormat {
 	var resp AttestationFormat
 	err := s.client.Call("Plugin.GetFormat", new(interface{}), &resp)
 	if err != nil {
-		log.Printf("ERROR during GetName RPC: %v\n", err)
+		log.Printf("ERROR during GetFormat RPC: %v\n", err)
 		return AttestationFormat_UNKNOWN_FORMAT
 	}
 	return resp
 }
 
-func (s *SchemeRPC) GetTrustAnchorID(token *AttestationToken) string {
+func (s *SchemeRPC) SynthKeysFromSwComponent(tenantID string, swComp *SwComponent) ([]string, error) {
+	var (
+		err  error
+		resp []string
+		args SynthKeysArgs
+	)
+
+	args.TenantID = tenantID
+
+	args.EndorsementJSON, err = json.Marshal(swComp)
+	if err != nil {
+		return nil, fmt.Errorf("failed unmarshaling response from RPC server: %w", err)
+	}
+
+	err = s.client.Call("Plugin.SynthKeysFromSwComponent", args, &resp)
+	if err != nil {
+		return nil, fmt.Errorf("failed call to SynthKeysFromSwComponent: %w", err)
+	}
+
+	return resp, nil
+}
+
+func (s *SchemeRPC) SynthKeysFromTrustAnchor(tenantID string, ta *TrustAnchor) ([]string, error) {
+	var (
+		err  error
+		resp []string
+		args SynthKeysArgs
+	)
+
+	args.TenantID = tenantID
+
+	args.EndorsementJSON, err = json.Marshal(ta)
+	if err != nil {
+		return nil, fmt.Errorf("failed unmarshaling response from RPC server: %w", err)
+	}
+
+	err = s.client.Call("Plugin.SynthKeysFromTrustAnchor", args, &resp)
+	if err != nil {
+		return nil, fmt.Errorf("failed call to SynthKeysFromTrustAnchor: %w", err)
+	}
+
+	return resp, nil
+}
+
+func (s *SchemeRPC) GetTrustAnchorID(token *AttestationToken) (string, error) {
 	data, err := json.Marshal(token)
 	if err != nil {
-		log.Printf("ERROR during token marshing: %v\n", err)
-		return ""
+		return "", err
 	}
 
 	var resp string
 	err = s.client.Call("Plugin.GetTrustAnchorID", data, &resp)
 	if err != nil {
-		log.Printf("ERROR getting trust anchor ID: %v\n", err)
-		return ""
+		return "", err
 	}
 
-	return resp
+	return resp, nil
 }
 
 func (s *SchemeRPC) ExtractEvidence(token *AttestationToken, trustAnchor string) (*ExtractedEvidence, error) {
-	var err error
-	var args ExtractEvidenceArgs
+	var (
+		err  error
+		args ExtractEvidenceArgs
+	)
```

Comment:
Code at line 282, 290 and 297 is inconsistent with the way it has been done on 248 and 253. Could you please align both of these properly?

Reply:
OK.  Done in 73da0276

Reply:
Done again in [a4a676f](https://github.com/veraison/veraison/pull/134/commits/a4a676f99a4772e35dc82083f527d642b09df89a)